smallcloudai/refact-chat-js
# WIP: Feat/user source

# User survey


## Description

Adds a user source survey.

## Type of change


## How to Test

- Step 1: Open chat
- Step 2:  Login (if not already)
- Step 3: observe a Survey.
- Step 4: dismiss the survey
- Step 5: logout
- Step 7: login
- Step 8: complete survey
- Step 9: logout
- Step 10: login
- Step 11: no survey.

## Screenshots (if applicable)


## Checklist


## Linked Issues
https://refact.fibery.io/Software_Development/UI-chat-js-133#Task/Login-Wizard-Extra-Step-334


## Additional Notes

I decided to use a dialog element to show the survey as the navigating away from a chat would lose the input (important for codelens and quick actions).
